iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/Analog.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface Analog {
+
+    enum Side {
+        ONE,
```

Comment:
I think you need a `NONE` side for loads, generators, etc.

Reply:
Is it not enough if it is `null` in this case?